service/raft/group0_state_machine.cc

```diff
@@ -199,16 +200,24 @@ future<> group0_state_machine::load_snapshot(raft::snapshot_id id) {
     _ss._topology_state_machine.event.broadcast();
 }
 
-future<> group0_state_machine::transfer_snapshot(gms::inet_address from, raft::snapshot_descriptor snp) {
+future<> group0_state_machine::transfer_snapshot(raft::server_id from_id, raft::snapshot_descriptor snp) {
+    auto from_ip = _address_map.find(from_id);
+    if (!from_ip.has_value()) {
+        // This is virtually impossible. We've just received the
+        // snapshot from the sender and must have updated our
+        // address map with its IP address.
+        const auto msg = format("Failed to apply snapshot from {}: ip address of the sender is not found", from_ip);
+        co_await coroutine::return_exception(raft::transport_error(msg));
+    }
```

Comment:
It looks like there is indentation problem here.

Reply:
Aligned the indentation better with the surrounding code.